include/vcpkg/base/messages.h

```diff
@@ -691,6 +699,22 @@ namespace vcpkg
                     (),
                     "This message is normally displayed only in development.",
                     "Downloading latest vcpkg-ce bundle...");
+    DECLARE_MESSAGE(DownloadRootsDir,
+                    (msg::env_var),
+                    "",
+                    "Specify the downloads root directory.\n(default: '{env_var}')");
+    DECLARE_MESSAGE(DuplicateCommandOption,
+                    (msg::command_name),
+                    "",
+                    "The option '{command_name}' can only be passed once.");
+    DECLARE_MESSAGE(DuplicateOptions,
```

Comment:
Note to self: double check that this is always a — thing…

Reply:
Should I change the name to specify the --?